.gitignore

```diff
@@ -21,3 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.eslintcache
```

Comment:
It would be a good idea to check how many of the files listed in here are actually present in your project. If they are not there (such as the coverage folder, `.env` files etc), they can be removed from this file